### PR TITLE
Feature/INV-25

### PR DESCRIPTION
**Pull request by jay-scott**
First draft of the Inventory API Apiary document

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1,33 +1,37 @@
...
 GENERATEASYNCCODE: TRUE
+GENERATEINTERFACES: TRUE
 USEMETHODPREFIX: TRUE
 USEMODELPOSTFIX: FALSE
 USECONTROLLERPOSTFIX: FALSE
 USEENUMPOSTFIX: FALSE
 USECONSTRUCTORSFORCONFIG: TRUE
-CSHARPDEFAULTNAMESPACE: DSI.SDK.<ApiName>
-JAVADEFAULTNAMESPACE: io.dudesolutions.<ApiName>
+CSHARPDEFAULTNAMESPACE: DSI.SDK.Event
```
Comment (aoffpo) [inline apiary.apib]: Inventory

File: apiary.apib [hunk 2]
```diff
@@ -1,33 +1,37 @@
...
+JAVADEFAULTNAMESPACE: io.dudesolutions.event
```
Comment (aoffpo) [inline apiary.apib]: inventory

File: apiary.apib [hunk 3]
```diff
@@ -59,19 +58,124 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
+
             X-Dude-Api-Limit: Unlimited or Number
             X-Dude-Api-Remaining: Unlimited or Number
             X-Dude-Api-Server: Api Server ID
-            
+
     + Attributes (Heartbeat)
 
+
+# Group InventoryItem
```
Comment (larastocks) [inline apiary.apib]: You can use spacing in the blueprint. Change to "Inventory Item" everywhere

File: apiary.apib [hunk 4]
```diff
@@ -59,19 +58,124 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
+## Base Collection Response (object)
++ `page`: 1 (number, required) - {INT} The current page provided in the response data.
++ `pageSize`: 1000 (number, required) - {INT} The page size returned in the response data.
++ `dataSize` (number, required) - {INT} The total records returned in the response.
++ `total` (number, required) - {INT} The total number of records available in the collection.
+
+## Base Client Object (object)
++ `id`: 1234567890 (number, required) - {INT} The ID of the API client.
+
+## Recurrence Pattern (object)
```
Comment (larastocks) [inline apiary.apib]: Can this be removed?

File: apiary.apib [hunk 5]
```diff
@@ -59,19 +58,124 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
+
+## System Links (object)
++ `endpoint` : `Inventory` (string, required) - The name of the endpoint.
++ `action` : `Get` (string, required) - The REST action.
++ `route` : `/accounts/{accountId}/inventoryitems` (string, required) - The route to the endpoint.
+
 ## Heartbeat (object)
-Represents a heartbeat response.
++ `cachevitalcheck`: true (boolean, required) - Vital check to ensure something is online.
++ `eventdbvitalcheck`: true (boolean, required) - Vital check to ensure something is online.
```
Comment (aoffpo) [inline apiary.apib]: inventorydbvitalcheck, also update comment to refer to database

File: apiary.apib [hunk 6]
```diff
@@ -59,19 +58,124 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
++ `links` (array[System Links], required) - The API routes.
+
+## System Links (object)
++ `endpoint` : `Inventory` (string, required) - The name of the endpoint.
++ `action` : `Get` (string, required) - The REST action.
++ `route` : `/accounts/{accountId}/inventoryitems` (string, required) - The route to the endpoint.
+
 ## Heartbeat (object)
-Represents a heartbeat response.
++ `cachevitalcheck`: true (boolean, required) - Vital check to ensure something is online.
```
Comment (aoffpo) [inline apiary.apib]: update comment to refer to cache

File: apiary.apib (hunk 4 above)
Comment (aoffpo) [inline apiary.apib]: remove this object

File: apiary.apib (hunk 1 above)
Comment (jay-scott) [reply apiary.apib]: Got it!

File: apiary.apib (hunk 3 above)
Comment (jay-scott) [reply apiary.apib]: Done

File: apiary.apib (hunk 5 above)
Comment (jay-scott) [reply apiary.apib]: Done.

File: apiary.apib (hunk 6 above)
Comment (jay-scott) [reply apiary.apib]: Done

File: apiary.apib (hunk 4 above)
Comment (jay-scott) [reply apiary.apib]: I'm assuming you mean the Base Client Object. If so, then it has been removed.

File: apiary.apib (hunk 3 above)
Comment (jay-scott) [reply apiary.apib]: Done.

File: apiary.apib (hunk 2 above)
Comment (jay-scott) [reply apiary.apib]: Done.

**Comment by aoffpo** [comment]
+1

File: apiary.apib [hunk 7]
```diff
@@ -59,19 +58,91 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
+    + Attributes (Inventory Item Update)
+
++ Response 200 (application/json; charset=utf-8)
+    + Attributes (Inventory Item Data)
+
+### DELETE
+
++ Response 204 (application/json; charset=utf-8)
+
 # Data Structures
```
Comment (larastocks) [inline apiary.apib:125]: All Data structures should have a description and a separate section for parameters. see standards documentation: https://readme.dudesoln.com/display/EA/Apiary+Documentation+Standards

File: apiary.apib [hunk 8]
```diff
@@ -59,19 +58,91 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
             X-Dude-Api-Remaining: Unlimited or Number
             X-Dude-Api-Server: Api Server ID
-            
+
     + Attributes (Heartbeat)
 
+
+# Group Inventory Item
+
+## Inventory Item Collection [/accounts/{accountId}/inventoryitems{?page,pagesize,orderby}]
```
Comment (larastocks) [inline apiary.apib:73]: Endpoints should have description. See https://readme.dudesoln.com/display/EA/Apiary+Documentation+Standards

File: apiary.apib [hunk 9]
```diff
@@ -59,19 +58,91 @@ Any response that does not provide a ```200 OK``` response code should be treate
...
+
+### POST
+
++ Request
+    + Attributes (Inventory Item Data)
+
++ Response 201 (application/json; charset=utf-8)
+    + Attributes (Inventory Item Update)
+
+## Inventory Item Detail [/accounts/{accountId}/inventoryitems/{id}]
```
Comment (larastocks) [inline apiary.apib]: Add Description

File: apiary.apib (hunk 7 above)
Comment (jay-scott) [reply apiary.apib:125]: Done.

File: apiary.apib (hunk 8 above)
Comment (jay-scott) [reply apiary.apib:73]: Done

**Comment by larastocks** [comment]
+1

**Pull request merged**